reposerver/repository/repository.go

```diff
@@ -1812,21 +1812,18 @@ var manifestFile = regexp.MustCompile(`^.*\.(yaml|yml|json|jsonnet)$`)
 
 // findManifests looks at all yaml files in a directory and unmarshals them into a list of unstructured objects
 func findManifests(logCtx *log.Entry, appPath string, repoRoot string, env *v1alpha1.Env, directory v1alpha1.ApplicationSourceDirectory, enabledManifestGeneration map[string]bool, maxCombinedManifestQuantity resource.Quantity) ([]manifest, error) {
-	absRepoRoot, err := filepath.Abs(repoRoot)
```

Comment:
why you changed it?

Reply:
i just moved this line below, to have our changes in one place (will simplify new upstream version merging)